modules/cpr/1.12.0/overlay/BUILD.bazel

```diff
@@ -0,0 +1,16 @@
+load("@rules_cc//cc:cc_library.bzl", "cc_library")
```

Comment:
I noticed the original repo actually supports Bazel, are you planning to upstream this change and update the doc?